### DISCUSSION TITLE
Support for Right-to-Left (RTL) Languages?

### DISCUSSION THREAD

**Discussion by brandonleichty** [beholdr/maska#214] [Q&A]
I currently have a mask that looks like this: `#### #### #### ####`

Let's say I have this number that I want to enter: 4111111111111111

And if I entered a number in a left-to-right (RTL) language, it'll look like `4111 1111 1111 1111` - which is correct and the expected results. 

However, if I have the `dir` direction set to `rtl`, the number is masking like this: `1111 1111 1111 4111`

Any idea how to address this? I've tried `data-masks-reverse`, but it didn't fix the issue. 

🙏

**Comment by ghost** [comment]
Hey, do you have some knowledge about right 2 left coding? Thank you!
